### DISCUSSION TITLE
Making the Min/Max and average only show values from the last hours_to_show

### DISCUSSION THREAD

**Discussion by Str00pwafel** [kalkih/mini-graph-card#760] [Ideas]
Hi,

Would it be possible to make it an option to only show the min/max and average values from hours_to_show?

Let's say I have a minimum value that is 2.00 in the last 24 hours(hours_to_show: 24), but after those 24 hours the minimum value in those 24h changed to 2.03 for example.

Right now it will still show 2.00 as minimum even though it's not visible in those last 24h.

I hope I made it clear what my idea is :D

**Comment by ildar170975** [comment]
The `hours_to_show` specifies time interval for drawing a graph: if "hours_to_show=24" and current datetime="02.01.22 14:00" then the graph will only include values from "01.01.22 14:00" till now.
And the extrema & average values are also defined for the same time interval: currently "max=2.0 (at 01.01.22 14:08)", and in 10 minutes (i.e. at 02.01.22 14:10) it may change to "max=1.9 (at 01.01.22 23:34)".
If you observed a different behaviour - it is a BUG imho.
